packages/client/src/components/pages/manageQuestionsPage/TableRowQuestions.tsx

```diff
@@ -34,8 +34,17 @@ const TableRowsQuestions: React.FC<ITableRowsQuestionsProps> = ({
         questions?.map((q) => [
           <tr key={q.id}>
```

Comment:
![image](https://user-images.githubusercontent.com/75366489/105755710-5998a680-5f54-11eb-99ae-754aa063017d.png)
I have not been able to solve this problem yet but it is worth trying to see why it makes this error

Reply:
i did not find a solution for this problem too 😭

Reply:
fixed this problem